recipes/dataframe/all/patches/remove-tests-bin-for-conan-package-v1.8.0.patch

```diff
@@ -0,0 +1,178 @@
+From 6733ba2a6a7b2691048c51be0b8817e9cdee53f3 Mon Sep 17 00:00:00 2001
```

Comment:
What's the difference between this file and 0001-remove-tests-bin-for-conan-package.patch? Could you re-use 0001-remove-tests-bin-for-conan-package.patch instead of this new one?

Reply:
No, I make the patch base on the release of DataFrame v1.8.0. I should use the same patch with v1.7.0, if there is no different, right?

Reply:
OK, I got it. It's right to re-use the 0001-remove-tests-bin-for-conan-package.patc, it there is no different.

Reply:
Yes, totally, if they are equal, you should use the same.